charts/steward/templates/clusterrole-aggregate.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: steward-aggregate-edit
+  labels:
+     {{- include "steward.labels" . | nindent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups:
+      - steward.sap.com
+    resources:
+      - tenants
+      - pipelineruns
+     verbs:
+      - create
+      - delete
+      - deletecollection
+      - get
+      - list
+      - patch
+      - update
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: steward-aggregate-view
+  labels:
+    {{- include "steward.labels" . | nindent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-view: "true"

```

Comment:
Shouldn't it be aggregated to edit and admin, too. Then we can leave out these permissions in `steward-aggregate-edit`.